cosmic-settings/src/pages/power/mod.rs

```diff
@@ -81,16 +80,16 @@ fn battery_info() -> Section<crate::pages::Message> {
             let battery_icon = widget::icon::from_name(page.battery.icon_name.clone());
             let battery_percent = text::body(format!("{}%", page.battery.percent));
 
-            let battery_time = text::body(if page.battery.remaining_duration > TimeDelta::zero() {
-                &page.battery.remaining_time
-            } else {
-                ""
-            });
+            let battery_time = text::body(page.battery.remaining_time());
 
             column::with_capacity(2)
                 .spacing(8)
                 .push(text::heading(&section.title))
-                .push(row!(battery_icon, battery_percent, battery_time).spacing(8))
+                .push(
+                    row!(battery_icon, battery_percent, battery_time)
+                        .align_items(Alignment::Center)
+                        .spacing(8),
```

Comment:
Looking at the designs, the spacing here should possibly be 4. The column spacing also seems smaller in the designs, but not sure exactly what value.